Source/WPEFramework/PluginServer.h

```diff
@@ -103,27 +107,79 @@ namespace PluginHost {
     private:
         class WorkerPoolImplementation : public Core::WorkerPool {
         private:
+#ifdef THUNDER_CRASH_HANDLER
+            struct requestStrListStruct
+            {
+                Core::IDispatch* iDispatchJob;
+                string requestString;
+
+                requestStrListStruct (Core::IDispatch* Job,  const std::string &rString)
+                {
+                    iDispatchJob = Job;
+                    requestString = rString;
+                }
+            };
+
+            typedef std::list<requestStrListStruct> RequestStrList;
```

Comment:
1) Risk of the iDispatchJob being dead. I did not find any cleanup upon revoke of the job.
2) std::vector for performance is applicable here. Do not thinkg you really need an std::list